### PR TITLE
Apple checkbox

### PR DESCRIPTION
**Pull request by SzymonRojek**
apple-checkbox

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -1,31 +1,62 @@
...
-    padding-top: 70px;
-    letter-spacing: 2px;
-    word-spacing: 4px;
+.is-visually-hidden {
+    position: absolute !important;
+    height: 1px;
+    width: 1px;
+    overflow: hidden;
+    clip: rect(1px 1px 1px 1px);
+    IE6,
```
Comment (Worie) [inline styles.css]: Remove those IE6/IE7 from here, this is not valid CSS. This should be placed as comment instead: `/** IE6 and IE7 support **/ 
Or remove this and line 48 completely

File: styles.css [hunk 2]
```diff
@@ -1,31 +1,62 @@
...
+    IE7 clip: rect(1px, 1px, 1px, 1px);
 }
 
-p {
-    font-size: 20px;
-    letter-spacing: 1px;
-    word-spacing: 1px;
+/*.custom-checkbox_inner:focus + .custom-checkbox {outline: 2px solid lightblue;}*/
```
Comment (Worie) [inline styles.css]: Please do not commit commented code. Do you support outline anyway?

File: styles.css [hunk 3]
```diff
@@ -1,31 +1,62 @@
...
+
+.custom-checkbox_inner:checked + .custom-checkbox {
+    background: #03a9f4;
+}
+
+/* apple green colour #8BCB71 is also good but I prefer blue*/
```
Comment (Worie) [inline styles.css]: This shouldn't really matter, stuff like those should not be commited (if you wanted to tell the person that will look and your code and review it, you can always leave a comment near each line that you want to discuss in PR)

File: styles.css (hunk 2 above)
Comment (Worie) [inline styles.css]: Please do not commit commented code. Do you support outline anyway?

File: styles.css (hunk 1 above)
Comment (Worie) [inline styles.css]: Remove those IE6/IE7 from here, this is not valid CSS. This should be placed as comment instead: `/** IE6 and IE7 support **/
Or remove this and line 48 completely, it's up to you

**Review by Worie — CHANGES_REQUESTED** [review]
Did you manage do to it all by yourself?

File: styles.css (hunk 3 above)
Comment (SzymonRojek) [reply styles.css]: hi, yes I did manage by myself, of course I watched a few trials on youtube - then I have spent time to watch again your lesson and finally I created a final project.

File: styles.css (hunk 2 above)
Comment (SzymonRojek) [reply styles.css]: yes I would like to support outline so I will uncomment it. thanks.

**Comment by SzymonRojek** [comment]
I would like to inform that I have seen your comments and I made all corrections in the code - branch apple-checkbox. Thank you so much.

**Pull request merged**